portfolio.html

```diff
@@ -0,0 +1,105 @@
+        <meta name="viewport contents" content="device-width, initial-scale=1.0">
+        <link rel ="stylesheet" type="text/css" href="portfolio-style.css">
+        <link rel ="stylesheet" type="text/css" href="normalise.css">
+    </head>
+    <body>
+        <header>
+
+            <section class="top-bar">
+                <div class="name">
+                    <h3>Matthew Hammond</h3>
+                </div>
+                <div class="top-links">
+                    <a href="#bio"><button type="button">Bio</button></a>
+                    <a href="#portfolio"><button type="button">Portfolio</button></a>
+                    <a href="#contact"><button type="button">Contact me</button></a>
+                </div>
+            </section>
+
+            <section class="landing">
+                <div class="landing-container">
+                    <img class="landing-image" src="./Assets/aerial.jpg">
+                    <div class="hero">
+                        <h1>Matthew Hammond</h1>
+                    </div>
+                    <div class="squire">
+                         <h1>Full-stack developer</h1>
+                    </div>
+                </div>
+            </section>
+            <!-------------------------------------------->
+            <section class="bio-container">
+                <div class="background" id="bio">
+                    <div class="sunset-container">
+                        <img class="sunset-image" src="./Assets/MOON.jpeg">
+                    </div>
+                        <div class="bio-title-container">
+                            <h1 class="Bio">Bio</h1>
+                        </div>
+                        <div class="paragraph-container">
+                            <p class="paragraph">
+                                Hey there. I'm currently learning to be a full-stack web-developer at Mayden academy.
+                                I consider my self a driven and motivated individual, who enjoys creative capacity and
+                                challenging my skills. I am always learning new things and adapting to succeed. I enjoy
+                                reading and excercising, comedy and coding. I enjoy working in teams, and am individually
+                                industrious.
+                            </p>
+                        </div>
+                    </div>
+            </section>
+            <!-------------------------------------------->
+            <section class="portfolio" id="portfolio">
+                        <div class="title-container">
+                            <h1 class="portfolio-title">Portfolio</h1>
+                        </div>
+                <div class="portfolio-container">
+                    <div class="container">
+                        <div class="grid">
+                            <div class="cell">
+                                <img src="./Assets/place-axolotl.png" class="responsive-image" />
```

Comment:
What happens when these are projects which you can describe? Where does the text go? Is there a link?